internal/currency/usd.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information
+
+package currency
+
+import (
+	"fmt"
+)
+
+// USD describes USD currency.
+type USD struct {
+	cents int
+}
+
+// Dollars converts dollars to USD amount.
```

Comment:
nit -- converts dollars to cents ?

Reply:
No, the "cents" is completely implementation detail, it could be changed to "microcents" without changing the public interface or comments.

Reply:
I think this might be a language thing

Reply:
cents are a specific thing microcents are not. I think it will be confusing to most American users but not a huge deal.